xla/xla.bzl

```diff
@@ -65,6 +65,7 @@ _XLA_SHARED_OBJECT_SENSITIVE_DEPS = if_static(extra_deps = [], otherwise = [
     Label("//xla/stream_executor/gpu:gpu_stream"),
     Label("//xla/stream_executor/rocm:all_runtime"),
     Label("//xla/stream_executor/rocm:stream_executor_rocm"),
+    "@tsl//tsl/util:determinism",
```

Comment:
I'm wondering if this is possible to narrow down to more specific targets. If not I'm happy to approve

Reply:
Yes, this is for linking MIOpen for rocm side.